client/src/components/SubstraitGraph.vue

```diff
@@ -85,6 +99,24 @@ export default {
       };
       image.src = svgUrl;
     },
+    async generateLink() {
+      try {
+        const resp = await axios.post("/api/save/", {
+          json_string: store.plan,
+          validation_levels: store.validation_override_levels,
+        });
+
+        const textField = document.createElement("textarea");
+        textField.innerText = this.currentUrl + "/plan/" + resp.data;
+        document.body.appendChild(textField);
+        textField.select();
+        document.execCommand("copy");
+        textField.remove();
```

Comment:
Is there a javascript library that hides this kind of workaround?

Reply:
using ClipboardJS now. Thanks for the suggestion.